source/elements/oneMKL/source/domains/blas/gemm.rst

```diff
@@ -45,6 +45,14 @@ op(``X``) = ``X``\ :sup:`H`,
        -  Ta 
        -  Tb 
        -  Tc 
+     * -  ``float`` 
```

Comment:
Can we match to DPC++ documentation to match the table titles for consistency? (Ta (A Matrices))....

Reply:
Sure, I've reordered the columns to match the DPC++ docs.

Reply:
i meant the table titles are different.

Reply:
Ah okay, updated ^